src/admin-orderable-add/orderable-add-ftaps-list.html

```diff
@@ -0,0 +1,50 @@
+<div class="modal" role="dialog" aria-hidden="true">
+    <div class="modal-dialog" style="width: 613px">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h1 class="modal-title">
+                    <h2>{{'adminOrderableEdit.associatedFtaps' | message}}</h2>
```

Comment:
Here a one time binding would be better to use. It can speed up the app performence:
{{:: 'adminOrderableEdit.associatedFtaps' | message}}
It can be applied for every non dynamic binding